### ISSUE TITLE
Syntax coloring when keyword escape and backslash character is used

### ISSUE THREAD

**Issue by cwolcott** [hankinsoft/SQLPro#981]
**Describe the bug**
When adding the reserved word ESCAPE '\' to  like statement the syntax coloring is messed up

**To Reproduce**
Review the following SQL Statements
```
SELECT * FROM MOLMAN.LIBRARIES WHERE DESCRIPTION like 'ABC_%' order by DESCRIPTION;

-- Using period as my escape character doesn't cause a problem with the syntax coloring.
SELECT * FROM MOLMAN.LIBRARIES WHERE DESCRIPTION like 'ABC._%' ESCAPE '.' order by DESCRIPTION;

-- Using backslash as my escape character causes a problem with the syntax coloring.
SELECT * FROM MOLMAN.LIBRARIES WHERE DESCRIPTION like 'ABC\_%' ESCAPE '\' order by DESCRIPTION;
```

**Expected behavior**
The "order by DESCRIPTION" in the 3rd statement should not be colored red.

**Screenshots**
<img width="755" alt="image" src="https://github.com/hankinsoft/SQLPro/assets/3777526/f35980aa-59bb-48f8-afe0-1d016aef5968">

**Environment details (please complete the following information):**
 - Device: MacBook Pro (2019)
 - OS: Sonoma 14.5
 - SQLPro app Version: SQLPro for Oracle Version 2023.81 (Build 581)
- Installation source: Directly from using since SQLPro Oracle updates can't be download from AppStore anymore.
- Target database server: Oracle

**Additional context**
None

## Issues will be closed until environmental details are provided.
If the above template is not completed, issues with be closed with the statement `Issue has been closed. Please edit the initial post (or create a new issue) and follow the template. Once completed, the issue may be reopened`.

**Comment by hankinsoft** [comment]
Coming for the next version:

<img width="1459" height="888" alt="Image" src="https://github.com/user-attachments/assets/c1779756-d302-413d-b2cd-f5fdf465743c" />

Also, not sure if I ever mentioned this but the iOS app sorts oracle now as well. (iPhone and iPad).

**Comment by cwolcott** [comment]
Just upgraded to Version 2026.50 (Build 111190.5) which was release 4 weeks ago.  So I guess the fix above is still in a future release.

**Comment by hankinsoft** [comment]
Can you grab the build mentioned here: https://github.com/hankinsoft/SQLPro/issues/1126#issuecomment-4327233360
And give this a try if/when you get the chance?

**Comment by cwolcott** [comment]
Yep SQLPro Studio Version 2026.87 (Build 111212) fixed this issue
